catalog-rest-service/src/main/java/org/openmetadata/catalog/events/ChangeEventHandler.java

```diff
@@ -72,10 +72,16 @@ public Void process(ContainerRequestContext requestContext, ContainerResponseCon
 
         // Add a new thread to the entity for every change event
         // for the event to appear in activity feeds
-        List<Thread> threads = getThreads(responseContext, changeEvent);
-        if (threads != null) {
-          for (var thread : threads) {
-            feedDao.create(thread);
+        // Skip creating threads for user or team related changes for now
+        if (!Entity.USER.equals(changeEvent.getEntityType()) && !Entity.TEAM.equals(changeEvent.getEntityType())) {
```

Comment:
Can we maintain a list of entities in Entity.java that we can grab with `shouldDisplayEntityChangeOnFeed` or something similar?

We need to avoid user, team, role, bot, policy

Reply:
Sounds good. I will work on updating the patch.